sound/soc/codecs/rt5682-sdw.c

```diff
@@ -718,6 +718,7 @@ static int rt5682_sdw_remove(struct sdw_slave *slave)
 
 static const struct sdw_device_id rt5682_id[] = {
 	SDW_SLAVE_ENTRY_EXT(0x025d, 0x5682, 0x1, 0, 0),
+	SDW_SLAVE_ENTRY_EXT(0x025d, 0x5682, 0x2, 0, 0),
```

Comment:
I don't think you need the 0x1, this is SoundWire 1.0. Can you check with Realtek if they support 1.0 devices?

Reply:
> I don't think you need the 0x1, this is SoundWire 1.0. Can you check with Realtek if they support 1.0 devices?

Thanks. Indeed, confirmed with Realtek, no SoundWire 1.0 rt5682 devices.